clinvar_ingest/slack.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import httpx
+
+import clinvar_ingest.config
+
+_logger = logging.getLogger("clinvar_ingest")
+
+
+def send_slack_message(message: str) -> None:
+    app_env = clinvar_ingest.config.get_env()
+    if not app_env.slack_token or not app_env.slack_channel:
+        _logger.warning(
+            "Both a slack channel and auth token are required. Slack messaging is disabled."
+        )
+    else:
+        slack_url = "https://slack.com/api/chat.postMessage"
+        data = {"text": message, "channel": app_env.slack_channel}
+        resp = httpx.post(
+            slack_url,
+            json=data,
+            headers={"Authorization": f"Bearer {app_env.slack_token}"},
+        )
```

Comment:
@toneillbroad since we have `requests` as a dependency, can we get rid of `httpx`? I think the `.post` function signature is the same